portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase

```diff
@@ -2047,6 +2047,67 @@ definition {
 			webContentTitle = "Web Content Title 2");
 	}
 
+	@description = "This is a use case for LPS-164933. Verify that it's possible to view the HTML Title and Description in SEO after changing the page name."
```

Comment:
Please extend the description with blocked message like 
https://github.com/gaborlovasdotcom/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase#L1464